src/Fields/regridding_fields.jl

```diff
@@ -40,7 +40,7 @@ output_field[1, 1, :]
 # output
 4-element OffsetArray(::Vector{Float64}, 0:3) with eltype Float64 with indices 0:3:
  0.0
- 2.333333333333333
+ 2.333333333333334
```

Comment:
👀

Reply:
Believe it or not, going from KernelAbstractions 0.8 down to 0.7 has this effect!

Reply:
hm!